### PR TITLE
Fix unit tests for Valkyrie

### PR DESCRIPTION
**Pull request by iamwolf**
This addresses and resolves https://github.com/openhumanoids/oh-distro/issues/22

### CODE REVIEW

**Comment by iamwolf** [comment]
Note: Both tests can now be run in a matrix using both default (valkyrie) and -v5 (working for both)

File: src/python/tests/testTeleopPanel.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def checkGraspFrame(inputGraspFrame, side):
 def onIkStartup(ikServer, startSuccess):
 
     side = 'left'
-    goalFrame = transformUtils.frameFromPositionAndRPY([0.5, 0.7, 1.2], [0, 90, -90])
+    goalFrame = transformUtils.frameFromPositionAndRPY([0.5, 0.5, 1.2], [0, 90, -90])
```
Comment (patmarion) [inline src/python/tests/testTeleopPanel.py]: I guess this is a goal frame that is reachable by both robots? nice!

**Comment by patmarion** [comment]
I think I updated src/python/tests/CMakeLists.txt to use -v5 for these tests.  Keeping that for now?

**Comment by iamwolf** [comment]
Didn't see that (https://github.com/RobotLocomotion/director/commit/7397749660b9a2d93cb0dbba53018258ab4da571). Happy to go with what you consider best. The changes here won't have an effect to current behaviour, rather allow being run for both v5 and val2. Is there a way to edit the CMakeLists to run these tests with the extra argument and without?

File: src/python/tests/testTeleopPanel.py (hunk 1 above)
Comment (iamwolf) [inline src/python/tests/testTeleopPanel.py]: Indeed. It returns 13 for Valkyrie

**Comment by patmarion** [comment]
I think the CMakeLists.txt uses a for loop that makes it tricky to repeat a test twice, but we could extend it to support this use case, would be nice to do.

**Comment by iamwolf** [comment]
I'm happy to have a go at it. What would be the best approach for this? Is expanding the add_python_test macro with a check whether extra_args is set, and if not adding it additionally with the extra argument v5, an ok solution or is there a preferable way in CMake?

**Comment by patmarion** [comment]
Right now the for loop is designed to add one test for each listed test file.  It {test name}_extra_args is defined then those args are added to the test command line.

Maybe we could design add_python_test() like this:

  - If {test name}_extra_args is defined, add it to the test command line
  - if {test name}_extra_args_2 is defined, add a 2nd test named {test name}_2 with the 2nd args
  - keep looping, check for {test name}_extra_args_{n} until the variable is no longer defined

So then we could have:

set(testTeleopPanel_extra_args -v5)
set(testTeleopPanel_extra_args_2 -val2)

And this would add two tests, one with -v5 and one with -val2.

**Pull request merged**